client/my-sites/checkout/src/components/wp-checkout.tsx

```diff
@@ -772,25 +772,59 @@ const CheckoutSummaryTitlePrice = styled.span`
 
 const CheckoutSummaryBody = styled.div`
 	box-sizing: border-box;
-	display: none;
 	margin: 0 auto;
 	max-width: 600px;
 	width: 100%;
 	padding: 24px;
+	${ hasCheckoutVersion( '2' )
+		? `grid-template-areas: 
+			"preview"
+			"review"
+			"summary"
+			"nudge"
+			"features";
+			display: grid;
```

Comment:
I may be misunderstanding but won't changing this from `display: none` to `display:grid` when `is-visible` is not applied cause this to be displayed when it shouldn't be? I guess I don't grok when this is supposed to be hidden or how it changes in v2.

Reply:
Yeah I saw a few changes that needed to be made and updated the PR - from my tests it correctly hides on mobile when `is-visible` is not applied.